eng/common/pipelines/templates/steps/get-pr-owners.yml

```diff
@@ -18,11 +18,8 @@ steps:
         --kusto-database-var KUSTO_DB `
         --kusto-table-var KUSTO_TABLE `
         --identity "$(Build.QueuedBy)"
+        --vsoVariable "${{ parameters.TargetVariable }}"
       $resolvedIdentity = $result[-1] | ConvertFrom-Json
```

Comment:
Do we even need to capture a result or resolvedIdentity?

Reply:
Nope! literally just was pushing a commit as you leave this comment :D